debs/bionic/archivematica-storage-service/Makefile

```diff
@@ -8,7 +8,7 @@ PACKBUILD_EXTRA_ARGS ?=
 #PACKBUILD_EXTRA_ARGS ?= "-b 1"
 PACKAGE = archivematica-storage-service
 BRANCH     ?= qa/0.x
-VERSION     ?= 0.13.0
+VERSION     ?= 0.13.1
```

Comment:
I think that the SS version should still be v0.13.0 (as that's the tag used to build the package), and we should use the release field for to indicate that this is the second build of the package. The final name should be something like "archivematica-storage-service_0.13.0-2"